.appveyor.yml

```diff
@@ -1,73 +1,52 @@
-shallow_clone: true
+version: 0.2.19.{build}
 
-platform:
-  # - x86
+#environment:
+
+platform: 
   - x64
 
-configuration:
-  - Debug
-  # - Release
+os: Visual Studio 2017
 
-image:
-  - Visual Studio 2015
-  # - Visual Studio 2017
+configuration: Release
 
-cache:
-  # - C:\cmake-3.10.1-win32-x86
-  # - C:\Program Files (x86)\IntelSWTools
-  # - C:\w_mkl_2018.0.124.exe
+init:
+  - git config --global core.autocrlf input
+  - echo %BLAS_LIBRARY%
 
-environment:
-  matrix:
-    # - BLAS_VERSION: Intel10_32
-    # - BLAS_VERSION: Intel10_64lp
-    - BLAS_VERSION: Intel10_64lp_seq
-    # - BLAS_VERSION: Intel
+clone_depth: 5
 
+skip_tags: true
 
-# init:
-#   - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+matrix:
+  fast_finish: false
+
+skip_commits:
+# Add [av skip] to commit messages
+  message: /\[av skip\]/
+
+environment:
+  global:
+    CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
+  matrix:
+    - BLAS_LIBRARY: openblas
+    - BLAS_LIBRARY: mkl
 
 install:
-  - ps: MD c:\logs\
-  - ps: |
-      if (![IO.File]::Exists("C:\cmake-3.10.1-win64-x64\bin\cmake.exe")) {
-        Start-FileDownload 'https://cmake.org/files/v3.10/cmake-3.10.1-win64-x64.zip'
-        7z x -y cmake-3.10.1-win64-x64.zip -oC:\
-      }
-      $env:PATH="C:\cmake-3.10.1-win64-x64\bin;$env:PATH"
-  - ps: |
-      if (![IO.File]::Exists("C:\w_mkl_2018.0.124.exe")) {
-        ECHO 'w_mkl_2018.0.124.exe does not exist. Preparing to install MKL ..'
-        Start-FileDownload 'http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/12079/w_mkl_2018.0.124.exe' -FileName 'C:\w_mkl_2018.0.124.exe'
-        C:\w_mkl_2018.0.124.exe -s -l c:\logs\extract-mkl.log -a install --eula=accept --components=all --output='C:\logs\install-mkl-output.txt' --installdir='C:\Program Files (x86)\IntelSWTools' --log='c:\logs\install-mkl.log' | Out-Null
-        ECHO 'Done'
-      }
-      $env:LD_MKL_PATH='C:\Program Files (x86)\IntelSWTools\compilers_and_libraries_2018.0.124\windows\mkl\lib\intel64_win\'
-      $env:LIB="$env:LD_MKL_PATH;$env:LIB"
-  - nuget install openblas -o C:\
+  - call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
+  - conda config --add channels conda-forge --force
+  - conda install --yes --quiet cmake ninja %BLAS_LIBRARY%
 
+  - call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x64
+  - set "LIB=%CONDA_INSTALL_LOCN%\Library\lib;%LIB%"
+  - set "INCLUDE=%CONDA_INSTALL_LOCN%\Library\include;%INCLUDE%"
 
 before_build:
-  - "ECHO %BLAS_VERSION%"
-  - "ECHO 'cmake -DBLA_VENDOR=%BLAS_VERSION% -G \"Visual Studio 14 2015 Win64\" .. '"
-  - ps: |
-      mkdir build
-      cd build
-  - cmake -DBLA_VENDOR=%BLAS_VERSION% -G "Visual Studio 14 2015 Win64" ..
+  - ps: if (-Not (Test-Path .\build)) { mkdir build }
+  - cd build
+  - cmake -G "Ninja" ..
```

Comment:
for my curiosity why using Ninja and not visual studio as generator?

Reply:
Ninja is much faster than msbuild.